applications/tari_base_node/windows/config/config.toml

```diff
@@ -0,0 +1,386 @@
+########################################################################################################################
```

Comment:
This should move to `common/config/presets/windows.toml`

Reply:
Moved